src/utils/file-upload-utils.js

```diff
@@ -11,10 +15,27 @@ const ALLOWED_FILE_EXTENSIONS = [
   "bmp",
   "ico",
 ]
+const defaultClient = CloudmersiveVirusApiClient.ApiClient.instance;
+
+// Configure API key authorization: Apikey
+const apikey = defaultClient.authentications.Apikey
+apikey.apiKey = CM_API_KEY
+
+const apiInstance = new CloudmersiveVirusApiClient.ScanApi()
+
+const scanFileForVirus = (fileBuffer) => new Promise((success, failure) => {
+    apiInstance.scanFile(fileBuffer, (error, data, response) => {
+      if (error) {
+        logger.error('Error when calling cloudmersive API')
```

Comment:
nit: "Cloudmersive API"

Reply:
We should also log the error, example: ``logger.error('Error when calling Cloudmersive API : ${error.message}`)`